hyrax.gemspec

```
@@ -31,7 +31,7 @@ SUMMARY
   # http://guides.rubyonrails.org/maintenance_policy.html
   spec.add_dependency 'rails', '~> 5.0'
 
-  spec.add_dependency 'active-fedora', '~> 13.0'
+  spec.add_dependency 'active-fedora', '~> 13.1.2' # pin for required bug fix to always rewind context
```

```suggestion
  spec.add_dependency 'active-fedora', '~> 13.1.2'
```
i think it's okay to just let the commit message and the fact of the pin speak for themselves here.